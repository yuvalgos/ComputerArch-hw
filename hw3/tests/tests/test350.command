./dflow_calc tests/test350.dat tests/test350.in p144 p60 p73 p158 p163 d198 p140 d12 d139 d60 p86 d109 p107 p188 d77 d124 d69 d107 p119 d23 d79 p91 d28 p184 p135 d110 p147 p78 d165 d64 p20 d32 d150 p35 d20 d49 d196 d190 p176 p91 p178 d59 p140 d166 d138 p4 p33 p169 d31 d24 d48 d89 d92 p81 d143 p151 p103 p72 p113 d177 d171 p45 p45 d87