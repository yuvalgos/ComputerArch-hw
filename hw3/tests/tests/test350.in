3 2 23 20
7 0 9 2
6 12 28 28
5 8 24 31
8 12 2 22
6 1 8 8
8 8 6 4
7 8 24 27
3 19 27 30
0 10 18 1
4 18 0 17
0 13 22 12
5 8 7 14
3 5 19 13
1 0 9 20
3 17 22 0
5 17 2 14
1 21 8 27
8 1 25 23
6 18 23 10
2 21 2 2
1 10 1 2
8 1 30 14
0 4 21 1
3 1 2 31
0 16 19 13
1 5 22 10
7 30 21 16
2 1 29 11
8 22 29 13
2 19 12 31
0 0 5 4
2 19 15 27
5 18 3 2
4 16 15 2
0 19 23 26
5 13 26 29
1 0 15 12
8 20 24 5
6 9 24 0
4 7 11 27
3 27 1 19
8 6 18 19
6 27 2 11
0 21 19 30
3 1 21 19
5 6 6 0
8 21 17 0
1 0 30 31
4 14 8 26
8 8 30 30
8 30 1 14
0 17 28 3
0 16 9 7
4 10 22 18
8 17 18 1
8 0 31 9
8 0 21 6
8 30 9 13
3 6 24 4
1 18 4 14
6 2 18 5
7 1 6 30
4 2 24 25
1 28 19 21
5 30 19 6
3 18 29 7
3 23 11 26
8 31 24 13
4 21 19 13
8 25 7 3
5 27 14 13
2 30 30 31
5 3 12 14
1 12 14 27
7 4 16 31
7 30 19 15
7 29 13 14
6 20 28 25
0 16 26 25
6 19 11 16
2 29 28 9
5 28 16 24
8 14 3 17
7 12 24 31
2 0 27 7
8 9 22 3
3 4 10 17
3 31 4 10
0 16 21 6
5 11 31 7
6 20 19 3
6 16 29 11
3 30 5 10
5 20 8 10
7 9 21 26
7 16 0 5
4 13 22 18
1 6 15 27
7 12 5 31
7 24 19 0
6 3 19 11
5 25 14 30
8 22 31 1
4 17 17 23
6 25 14 5
2 12 29 11
3 9 26 15
7 8 7 6
1 10 20 31
5 3 5 31
0 11 31 30
5 31 9 21
5 3 7 1
2 6 20 5
0 14 14 29
7 5 2 4
7 15 15 9
0 24 20 18
5 10 15 28
7 5 4 29
3 27 8 1
4 23 6 18
5 28 5 9
3 11 6 20
2 28 21 25
4 6 12 25
0 2 30 19
5 1 15 5
5 15 16 8
2 2 18 8
0 5 1 12
6 3 15 25
8 3 19 21
2 31 6 7
8 14 12 23
4 16 11 11
0 21 19 2
4 25 17 8
0 21 9 16
8 5 20 8
7 6 5 16
8 14 8 24
8 11 15 22
3 13 26 6
6 29 12 26
1 31 25 3
7 22 26 7
4 12 3 17
4 26 29 8
8 12 31 5
6 5 22 18
2 5 11 5
2 4 25 11
7 10 2 12
7 20 1 30
2 13 21 27
0 23 3 1
8 22 11 1
7 10 9 9
1 14 15 28
0 12 31 31
7 27 26 17
3 13 21 1
3 28 0 23
5 28 19 0
4 0 5 22
8 20 9 8
1 10 25 24
7 24 3 1
0 2 14 1
5 24 19 7
7 25 30 7
1 18 17 10
7 10 6 21
3 6 4 18
7 14 28 2
8 0 27 18
5 23 15 13
7 31 23 12
3 6 16 5
0 9 23 23
8 27 0 30
6 15 7 31
4 12 13 27
0 25 25 22
1 31 30 5
8 5 24 26
8 14 0 11
7 4 15 21
1 15 21 15
7 17 27 10
5 11 4 3
2 13 22 1
3 19 28 9
4 19 10 3
8 29 25 31
6 23 19 31
7 23 5 23
4 5 9 18
