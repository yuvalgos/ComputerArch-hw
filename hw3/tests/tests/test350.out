Reading the opcodes latency info from tests/test350.dat ... Got latency for 9 opcodes
Reading the program file tests/test350.in ... Found 200 instructions
getProgDepth()==229
getDepDepth(144)==131
getDepDepth(60)==25
getDepDepth(73)==30
getDepDepth(158)==145
getDepDepth(163)==124
getInstDeps(198)=={187,197}
getDepDepth(140)==89
getInstDeps(12)=={-1,-1}
getInstDeps(139)=={107,136}
getInstDeps(60)=={23,49}
getDepDepth(86)==55
getInstDeps(109)=={94,88}
getDepDepth(107)==0
getDepDepth(188)==187
getInstDeps(77)=={36,49}
getInstDeps(124)=={114,94}
getInstDeps(69)=={35,36}
getInstDeps(107)=={-1,-1}
getDepDepth(119)==49
getInstDeps(23)=={20,22}
getInstDeps(79)=={-1,70}
getDepDepth(91)==53
getInstDeps(28)=={-1,-1}
getDepDepth(184)==210
getDepDepth(135)==117
getInstDeps(110)=={26,88}
getDepDepth(147)==11
getDepDepth(78)==59
getInstDeps(165)=={80,85}
getInstDeps(64)=={35,47}
getDepDepth(20)==7
getInstDeps(32)=={-1,-1}
getInstDeps(150)=={146,140}
getDepDepth(35)==0
getInstDeps(20)=={0,0}
getInstDeps(49)=={12,-1}
getInstDeps(196)=={185,186}
getInstDeps(190)=={139,183}
getDepDepth(176)==135
getDepDepth(91)==53
getDepDepth(178)==131
getInstDeps(59)=={-1,23}
getDepDepth(140)==89
getInstDeps(166)=={152,158}
getInstDeps(138)=={104,108}
getDepDepth(4)==7
getDepDepth(33)==7
getDepDepth(169)==124
getInstDeps(31)=={26,23}
getInstDeps(24)=={0,-1}
getInstDeps(48)=={27,-1}
getInstDeps(89)=={69,59}
getInstDeps(92)=={81,90}
getDepDepth(81)==59
getInstDeps(143)=={129,103}
getDepDepth(151)==30
getDepDepth(103)==99
getDepDepth(72)==45
getDepDepth(113)==92
getInstDeps(177)=={162,173}
getInstDeps(171)=={80,40}
getDepDepth(45)==55
getDepDepth(45)==55
getInstDeps(87)=={54,55}
